src/ucs/sys/sock.c

```diff
@@ -237,3 +237,152 @@ const char* ucs_sockaddr_str(const struct sockaddr *sock_addr,
 
     return str;
 }
+
+static ucs_status_t ucs_sockaddr_cmp_check_family(const struct sockaddr *sa)
```

Comment:
looks like this function can be moved inside `ucs_sockaddr_cmp_family_equal`.
So finally we would have just one function for check families.

Reply:
this is not needed anymore